app/geant-exporter/PhysicsList.cc

```diff
@@ -42,16 +81,217 @@ void PhysicsList::ConstructParticle()
+ * | Ionization                   | G4MollerBhabhaModel       |
+ * | Bremsstrahlung (low E)       | G4SeltzerBergerModel      |
+ * | Bremsstrahlung (high E)      | G4eBremsstrahlungRelModel |
+ * | Coulomb scattering           | G4eCoulombScatteringModel |
+ * | Multiple scattering (low E)  | G4UrbanMscModel           |
+ * | Multiple scattering (high E) | G4WentzelVIModel          |
+ *
+ * \note
+ * - Bremsstrahlung models are selected manually at compile time using
+ *   \c BremsModelSelection and need to be updated accordingly.
+ * - Coulomb and multiple scatterings are currently disabled.
+ */
+void PhysicsList::add_e_processes()
+{
+    auto       physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
+    const auto electron     = G4Electron::Electron();
+    const auto positron     = G4Positron::Positron();
+
+    if (true)
+    {
+        // e+e- annihilation: G4eeToTwoGammaModel
+        physics_list->RegisterProcess(new G4eplusAnnihilation(), positron);
+        CELER_LOG(info) << "Loaded pair annihilation with G4eplusAnnihilation";
+    }
+
+    if (true)
+    {
+        // e-e+ ionization: G4MollerBhabhaModel
+        auto electron_ionization = std::make_unique<G4eIonisation>();
+        auto positron_ionization = std::make_unique<G4eIonisation>();
+        electron_ionization->SetEmModel(new G4MollerBhabhaModel());
+        positron_ionization->SetEmModel(new G4MollerBhabhaModel());
+        physics_list->RegisterProcess(electron_ionization.release(), electron);
+        physics_list->RegisterProcess(positron_ionization.release(), positron);
+
+        CELER_LOG(info) << "Loaded e-e+ ionization with G4MollerBhabhaModel";
+    }
+
+    if (true)
+    {
+        // Bremsstrahlung: G4SeltzerBergerModel + G4eBremsstrahlungRelModel
+        // Currently only using Seltzer-Berger
+        auto models = BremsModelSelection::seltzer_berger;
+
+        auto electron_brems = std::make_unique<BremsstrahlungProcess>(models);
+        auto positron_brems = std::make_unique<BremsstrahlungProcess>(models);
+        physics_list->RegisterProcess(electron_brems.release(), electron);
+        physics_list->RegisterProcess(positron_brems.release(), positron);
+
+        switch (models)
+        {
+            case BremsModelSelection::seltzer_berger:
+                CELER_LOG(info) << "Loaded Bremsstrahlung with "
+                                   "G4SeltzerBergerModel";
+                break;
+            case BremsModelSelection::relativistic:
+                CELER_LOG(info) << "Loaded Bremsstrahlung with "
+                                   "G4eBremsstrahlungRelModel";
+                break;
+            case BremsModelSelection::all:
+                CELER_LOG(info)
+                    << "Loaded Bremsstrahlung with "
+                       "G4SeltzerBergerModel and G4eBremsstrahlungRelModel";
+                break;
+            default:
+                CELER_LOG(warning) << "Bremsstrahlung model selection unknown";
```

Comment:
We only have enums for the first three, so you should change this to
```suggestion
                CELER_ASSERT_UNREACHABLE();
```